openfisca_france_local/departements/eure_et_loir/adefip.py

```diff
@@ -0,0 +1,15 @@
+ # -*- coding: utf-8 -*-
+from openfisca_france.model.base import Variable, Individu, MONTH
+
+
+class eure_et_loir_eligibilite_adefip(Variable):
```

Comment:
@guillett Pour la référence législative, tu pourrais ajouter le lien vers le pdf (vu que tu ajoutes les fichiers directement dans le dépôt tu as une url).

Reply:
Pour la première itération, je pense que ce lien est superflu. L'Eure-et-Loir semble voir intégrer plusieurs aides et cela pourrait les amener à travailler sur un dépôt dédié et le lien se retrouverait cassé. Cette absence te parait-elle bloquante ?

Reply:
Non non pas du tout bloquante, juste une idée.